connectors/helidon-connector/pom.xml

```diff
@@ -42,7 +42,7 @@
         <dependency>
             <groupId>io.helidon.jersey</groupId>
             <artifactId>helidon-jersey-connector</artifactId>
-            <version>3.0.0-M1</version>
+            <version>3.0.1</version>
```

Comment:
We do not have CQ for 3.0.1

Reply:
[CQ24261](https://dev.eclipse.org/ipzilla/show_bug.cgi?id=24261)